### PR TITLE
Send auth mode telemetry events

### PR DESCRIPTION
**Pull request by mvanchaa**

### CODE REVIEW

File: src/AzureAuth/Program.cs [hunk 1]
```diff
@@ -31,7 +31,8 @@ private static void Main(string[] args)
             // trigger telemetry before we ever get to disable it.
             //
             // To disable telemetry a user need only leave this environment variable unset. It's off by default.
-            string applicationInsightsIngestionToken = Environment.GetEnvironmentVariable(EnvVars.ApplicationInsightsIngestionTokenEnvVar);
+            // string applicationInsightsIngestionToken = Environment.GetEnvironmentVariable(EnvVars.ApplicationInsightsIngestionTokenEnvVar);
+            string applicationInsightsIngestionToken = "2ba47d2c-65c4-4bf9-a77a-f8ffd05cbafe";
```
Comment (goagain) [inline src/AzureAuth/Program.cs]: Is this an internal token? If yes, delete this branch as soon as possible. It's too dangerous.

**Review by goagain — CHANGES_REQUESTED** [review]

**Comment by shalinikhare27** [comment]
@mvanchaa - I would first break this PR into multiple PRs for easy review. :)

**Comment by shalinikhare27** [comment]
Also, side note, I am implementing IWA as a auth flow. So depending on who's PR gets merged first, one of us will need to add telemetry in IWA as well. :)

**Comment by mvanchaa** [comment]
> Also, side note, I am implementing IWA as a auth flow. So depending on who's PR gets merged first, one of us will need to add telemetry in IWA as well. :)

@shalinikhare27 Yup, I was planning to add those changes to IWA after yours is merged.

**Comment by mvanchaa** [comment]
> @mvanchaa - I would first break this PR into multiple PRs for easy review. :)

Thanks for the suggestion @shalinikhare27! That will make things easy for me too :)

**Pull request closed without merging**